framework/include/timeintegrators/ActuallyExplicitEuler.h

```diff
@@ -0,0 +1,68 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef ACTUALLYEXPLICITEULER_H
+#define ACTUALLYEXPLICITEULER_H
+
+#include "TimeIntegrator.h"
+#include "MeshChangedInterface.h"
+
+#include "libmesh/linear_solver.h"
+
+// Forward declarations
```

Comment:
I assume people that look at our code know C++. I think they are familiar with forward declarations, so this does not add any value.

Reply:
We have this all over and I like it - just some organization